swagger.json

```diff
@@ -1641,7 +1641,87 @@
                 }
             }
         },
-      "/fleet/vehicles/locations": {
+        "/fleet/vehicles/{vehicle_id_or_external_id}": {
+            "get": {
+                "tags": [
+                    "Fleet"
+                ],
+                "summary": "/fleet/vehicles",
+                "description": "Gets a specific vehicle.",
+                "operationId": "getFleetVehicle",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "$ref": "#/parameters/vehicleIdOrExternalId"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "The specified vehicle.",
+                        "schema": {
+                            "$ref": "#/definitions/FleetVehicleResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+                "tags": ["Fleet"],
+                "summary": "/fleet/vehicles",
+                "description": "Updates the specified vehicle",
+                "operationId": "patchFleetVehicle",
+                "parameters": [
+                    {
+                        "$ref": "#/parameters/accessTokenParam"
+                    },
+                    {
+                        "$ref": "#/parameters/vehicleIdOrExternalId"
+                    },
+                    {
+                        "in": "body",
+                        "name": "data",
```

Comment:
Should we use the name `data` here, or `patchedVehicle`?

Reply:
It only seems to matter for how you use the autogenerated client.  I was wavering between data or body, so that querying the endpoint is something like 
```
apiClient.Fleet.PatchFleetVehicle(fleet.NewPatchFleetVehicleParams().
    WithAccessToken(...).
    WithVehicleIDOrExternalID(...).
    WithData(...<body params>...))
```
but as to what it should be it really depends on how we want people to use the client.  I prefer something more regular for adding to the body, so `data` or `body`, but if we have an existing convention then it makes sense to change it to `patchedVehicle`.

Reply:
SGTM!